static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -744,14 +742,20 @@ export class TraceTreeNode<T extends TraceTree.NodeValue> {
     }
   }
 
-  clone(): TraceTreeNode<T> {
+  deepClone(): TraceTreeNode<T> {
     const node = new TraceTreeNode(this.parent, this.value, this.metadata);
     node.expanded = this.expanded;
     node.zoomedIn = this.zoomedIn;
     node.canFetchData = this.canFetchData;
     node.space = this.space;
-    node.children = this.children;
-    node.invalidate(node);
+    node.metadata = this.metadata;
+
+    for (const child of this.children) {
```

Comment:
This isnt really "deep" right, you are just cloning the first level of children. Why is that?